data-serving/scripts/setup-db/src/index.ts

```diff
@@ -43,6 +48,14 @@ const setupDatabase = async ({
 
         const collection = await database.getCollection(collectionName);
 
+        print('Dropping indexes👇');
+        const indexName = `${collectionName}Idx`;
+        await collection.dropIndex(indexName);
```

Comment:
I don't think you need to drop the indexes because we drop the collection if it exists but I guess it can't hurt

Reply:
yeah I figured it wouldn't hurt.